docs/getting-started.md

```diff
@@ -78,9 +78,13 @@ You can now run a mining node:
 radicle-registry-node --miner <address>
 ```
 
-`TODO: You can see mining in the logs`
+If your node successfully mined a block you will see a log line like this
+```
+15:50:13.041 INFO radicle_registry_node::service  Imported own block #1322 (0xefd1…4445)
+```
+Depending on how lucky you are mining a block might take minutes or hours.
```

Comment:
```suggestion
Depending on how lucky you are, mining a block might take minutes or hours.
```

I would prefer to be a bit more scientific though :P what will affect that _luck_? A faster machine, more memory? We should mention that (instead).

Reply:
It really is luck: Since only one core is mining and nobody uses specialized hardware all miners have roughly the same probability of mining a block.

Reply:
ah I see! lol Advanced stuff TM.